### ISSUE TITLE
Getting chat fails if account has chats with multiple characters

### ISSUE THREAD

**Issue by K0shachiyamyatA** [kramcat/CharacterAI#51]
https://beta.character.ai/chat2?char=gIor6bOT0GtU7tTNvAvc1NxAW5O7AyqdBx-cApcH9wk

if account has multiple characters chats(?) i think
the link will have `chat2` instead of `chat`. And this will not allow API to get character`s history.
```py
        async def get_chat(
            self, char: str = None, *,
            wait: bool = False, token: str = None,
            **kwargs
        ):
            return await _PostResponse(
                post_link='chat/history/continue/',
                data={
                    'character_external_id': char,
                    **kwargs
                },
                wait=wait, token=token
            )
```

**Comment by K0shachiyamyatA** [comment]
UPD: manually removing `2` from the link and starting the chat like that seems to fix the issue. But i would like the library to handle this case for me.

**Comment by kramcat** [comment]
chat2 is a different version of chat, which the library does not support yet

**Comment by Psyk0loge** [comment]
> chat2 is a different version of chat, which the library does not support yet

is there a way to change the chat used in characterai or is the library not working at the moment?

**Comment by K0shachiyamyatA** [comment]
> > chat2 is a different version of chat, which the library does not support yet
> 
> is there a way to change the chat used in characterai or is the library not working at the moment?

Check my previous comment.

**Comment by Psyk0loge** [comment]
okay, I see that removing the number manually will work, but automatically it does not work yet right?

**Comment by kramcat** [comment]
fixed on v0.8.0
